applications/crossbar/priv/couchdb/schemas/accounts.json

```diff
@@ -175,12 +175,18 @@
       },
       "default": {}
     },
-        "language": {
-            "type": "string",
-            "required": false,
-            "name": "Language",
-            "description": "The language for this account",
-            "default":"en-us"
-        }
+    "language": {
+      "type": "string",
+      "required": false,
+      "name": "Language",
+      "description": "The language for this account",
+      "default":"en-us"
+    },
+    "org": {
```

Comment:
org is a full legal name of the organization. Names (as practice shows) are usually shortened or even can be not connected to organization name in any possible way